<launch>
    <!-- DEBUG: launch-prefix="gdb -ex run -(delete)-args" -->
    <node name="bitcq_camera" pkg="bitcq_camera" type="bitcq_camera_node" output="screen"> 
        <!--device_type: gmsl  usb  rtsp  hik_sdk dahua_sdk-->
        <param name="device_type" value="dahua_sdk"/>
        <!--device_address: device's hardware io address : "/dev/video0" "rtsp://...." "192.168.1.1"-->
        <param name="device_address" value="/dev/video0"/>
        <param name="image_width" value="1920" />
        <param name="image_height" value="1080" />
        <param name="fps" value="30"/>
        <!--pixel_format: yuyv uyvy mjpeg yuvmoni10 rgb24 grey other-->
        <param name="pixel_format" value="uyvy" />
    </node>
    <node name= "image_view" pkg= "image_view" type= "image_view" respawn= "false" output= "screen" >
     <remap from= "image" to= "/bitcq_camera/image_source" />
     <param name= "autosize" value= "true" />
   </node>
</launch>